test/e2e/data/shared/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
```

Comment:
why do we need this ?

Reply:
To make it aware of the versioning scheme in the latest CAPI nightly builds that we are pulling in.